src/main/java/kafka/Producer.java

```diff
@@ -39,7 +39,6 @@ private Headers getHeaders(ConsumerRecord<String, String> record) {
         return headersToSend;
     }
 
-    // Source: https://github.com/1and1/reactive/blob/e582c0bdbfb4ab2a0780c77419d0d3ee67f08067/reactive-kafka/src/main/java/net/oneandone/reactive/kafka/CompletableKafkaProducer.java#L42
```

Comment:
No, don't remove that. We can't take code from someone else without proper attribution

Reply:
ok